### PR TITLE
Merge template version 2.14.1

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Merge template version 2.14.1

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] `CHANGELOG.md` is updated.

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -179,41 +179,6 @@
...
-                    "fastq_align_chromap": {
-                        "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
-                        "installed_by": ["subworkflows"]
-                    },
-                    "fastq_fastqc_umitools_trimgalore": {
-                        "branch": "master",
-                        "git_sha": "cfd937a668919d948f6fcbf4218e79de50c2f36f",
-                        "installed_by": ["subworkflows"]
-                    },
```
Comment (maxulysse) [inline modules.json:216]: some subworkflows got deleted here, is that expected?

**Review by maxulysse — COMMENTED** [review]
I'd say you have some delete subworkflows in the `modules.json`, but apart from that, looks ok to me

File: modules.json (hunk 1 above)
Comment (JoseEspinosa) [reply modules.json:216]: Good spot! Probably, I deleted them by mistake when resolving the merge conflicts, I'll take a look

**Comment by JoseEspinosa** [comment]
> I'd say you have some delete subworkflows in the `modules.json`, but apart from that, looks ok to me

Thanks @maxulysse!  I think it is fixed now will be great if you could take a look.

**Review by maxulysse — APPROVED** [review]

**Pull request merged**